tools/cli/internal/cli/changelog/create.go

```diff
@@ -81,6 +125,9 @@ func CreateBuilder() *cobra.Command {
 		Short:   "Generate the changelog for the OpenAPI spec.",
 		Args:    cobra.NoArgs,
 		RunE: func(_ *cobra.Command, _ []string) error {
+			if err := opts.validations(); err != nil {
```

Comment:
[q] why validations in RunE and not `PreRunE`?

Reply:
updated